sources/packages/web/src/views/student/application-offering-change-request/ActiveApplicationOfferingDetails.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <body-header-container>
+    <template #header>
+      <body-header :title="changeRequest.studentFullName">
```

Comment:
https://github.com/bcgov/SIMS/pull/2196/files#r1298762496

Reply:
@ann-aot I cannot find this link. Is the comment deleted?

Reply:
https://github.com/bcgov/SIMS/pull/2196#discussion_r1306239201